nca/Utils/CmdlineRunner.py

```diff
@@ -95,7 +98,7 @@ def get_k8s_resources(resource):
         cmdline_list = ['kubectl', 'get', resource, '-o=json']
         if resource in ['networkPolicy', 'authorizationPolicy', 'pod', 'ingress', 'Gateway', 'VirtualService', 'sidecar']:
             cmdline_list.append('--all-namespaces')
-        return CmdlineRunner.run_and_get_output(cmdline_list)
+        return CmdlineRunner.run_and_get_output(cmdline_list, check_for_silent_exec=True)
```

Comment:
Shouldn't also be set True from `get_calico_resources`, not only from `get_k8s_resources `?

Reply:
currently if the topology section is missing / or  user runs cmdline without resources or without policies - our default is to load missing resources from k8s live cluster - so it is the only relevant case for a silent attempt

we load from calico only if the input is calico - a provided "live cluster" input does not ignore errors for any of the layers